ceph.spec.in

```diff
@@ -820,6 +820,23 @@ mkdir -p %{_localstatedir}/run/ceph/
 %config %{_sysconfdir}/bash_completion.d/rbd
 %config(noreplace) %{_sysconfdir}/ceph/rbdmap
 %{python_sitelib}/ceph_argparse.py*
+%if 0%{?rhel} >= 7 || 0%{?fedora}
+/usr/lib/udev/rules.d/50-rbd.rules
+%else
```

Comment:
rbd.rules continues to be listed in the base package files list:
 732 %if 0%{?debian}
 733 /lib/udev/rules.d/50-rbd.rules
 734 %endif
 735 %if 0%{?rhel} >= 7 || 0%{?fedora} || 0%{?suse_version}
 736 /usr/lib/udev/rules.d/50-rbd.rules
 737 %endif
This should be removed.

Also as Owen indicated, the %else condition doesn't look right, as it'll be triggered for SUSE distros in addition to the %if 0%{?suse_version} logic that you added.